test/ossfuzz.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -eu
+# This script is meant to be run by
+# https://github.com/google/oss-fuzz/blob/master/projects/utf8proc/Dockerfile
+
+mkdir build
+cd build
+cmake .. -DUTF8PROC_ENABLE_TESTING=ON
+make -j$(nproc)
+
+
+$CC $CFLAGS -I$SRC/utf8proc \
+    $SRC/utf8proc/test/fuzzer.c \
+    -o $OUT/utf8proc_fuzzer \
+    $LIB_FUZZING_ENGINE $SRC/utf8proc/build/libutf8proc.a
```

Comment:
Shouldn't this be built by cmake, e.g. with a `-DLIB_FUZZING_ENGINE="$LIB_FUZZING_ENGINE" flag?

Reply:
It will now link against `$LIB_FUZZING_ENGINE` when defined before falling back to the standalone driver.